packages/clarity-decode/types/layout.d.ts

```diff
@@ -17,9 +17,10 @@ export import Constant = Layout.Constant;
 export interface DomData {
     id: number;
     parent: number;
-    next: number;
+    previous: number;
     tag: string;
     position: number;
     attributes?: Layout.Attributes;
     value?: string;
+    next?: number; /* deprecated since v0.4.5 */
```

Comment:
>next [](start = 4, length = 4)

This isn't backwards compatible, is it? Should this be 0.5 instead?

Reply:
It is backward compatible.